New API & Update Properties for Anomaly Detector

juaduan opened the pull request:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [x] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  December 2021
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  December 2021
  4. If updating an existing version, please select the specific langauge SDKs and CLIs that must be refreshed after the swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [x] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that label “WaitForARMFeedback” will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits.

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

mengaims committed:
add impute strategy

mengaims committed:
Add severity definition

juaduan committed:
Merge pull request #1 from juaduan/uvad-update

Uvad update

Commit:
update MultivariateAnomalyDetector.json

Commit:
update examples

juaduan committed:
Merge pull request #2 from juaduan/juaduan/dev-cognitiveservices-AnomalyDetector-v1.1-preview

Juaduan/dev cognitiveservices anomaly detector v1.1 preview

Commit:
Merge branch 'main' of https://github.com/juaduan/azure-rest-api-specs

Commit:
update swagger

Commit:
update path

juaduan committed:
Merge pull request #3 from juaduan/juaduan/mvad_swagger

Juaduan/mvad swagger

Commit:
update example

Commit:
update result structure

Commit:
update

Commit:
update...

Commit:
change version

tjprescott commented:
@juaduan I am confused by this PR. The title says "update example". The purpose section of the checklist says "update existing version for new feature". It looks like neither of these. It looks like a new API version. Can you please clarify?

Also, please address questions 2 and 3 in the checklist.

Commit:
delete test files

juaduan commented:
> @juaduan I am confused by this PR. The title says "update example". The purpose section of the checklist says "update existing version for new feature". It looks like neither of these. It looks like a new API version. Can you please clarify?
> 
> Also, please address questions 2 and 3 in the checklist.

@tjprescott I have updated the title. We are going to add a new API and add new properties to the response of an API. We plan to release the service in December 2021 and I have updated questions 2&3. Thanks.

Commit:
reset version

Commit:
update last detect parameters

Commit:
fix break changes

Commit:
fix errors

Commit:
update examples

juaduan committed:
Merge branch 'main' into juaduan/mvad_swagger_latest

Commit:
Revert "Merge branch 'main' into juaduan/mvad_swagger_latest"

This reverts commit 9524fe794fb190bb7aea196d954203c55604efc5, reversing
changes made to fef9721866e175d220def5ddfff69e15ce39add5.

tjprescott commented:
@juaduan thank you for the clarification. As the bot states, adding new APIs and changing the response type of an existing API requires a new API Version. You would need to get archboard approval for this.

juaduan commented:
move to another pr that has both mvad & uvad api changes.
